sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -34,7 +34,13 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
         path: AppRoutes.LoginWithBusinessBCeID,
         name: InstitutionRoutesConst.LOGIN_WITH_BUSINESS_BCEID,
         component: Login,
-        props: { showBasicBCeIDMessage: true },
+        props: { showBasicBCeIDMessage: true, showBasicDisabledUserMessage: false },
```

Comment:
I believe that "Basic" in this context doesn't make too much sense. I would recommend just to just "showDisabledUserMessage".